src/Containers/FeaturesProfileCard/FeaturedProfilesCard.stories.tsx

```diff
@@ -87,15 +87,26 @@ const profiles = [
 export default {
     title: createStoryTitle('Featured Profiles Card'),
     component: FeaturedProfilesCard,
-    args: {
-        profileData: profiles,
-        alt: 'Profile Image',
-        counterBool: false,
-    },
 } as Meta;
 
-export const Demo: Story<IFeaturedProfilesCardProps> = (args) => (
+const Template: Story<IFeaturedProfilesCardProps> = (args) => (
     <FeaturedProfilesCard {...args} />
 );
 
-//todo different use cases presets.
\ No newline at end of file
+export const Basic = Template.bind({});
+Basic.args = {
+    profileData: profiles,
+    counterBool: false,
```

Comment:
Don't put Types in the Name

We dont say NameString, we just say Name.

Rename this to isCounterShowing or isCounterDisplayed or isProfileCountShowing

the prefix is denotes a Boolean value